components/Tabs/Tabs.js

```diff
@@ -2,44 +2,52 @@
 class TabsItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
+    this.element.classList.add('Tabs__item-selected');
   }
 
   deselect() {
     // should use classList
+    this.element.classList.remove('Tabs__item-selected');
   }
 }
 
 class TabsLink {
```

Comment:
Hm, I don't know if you were able to complete it all the way but when I click on the tab boxes, they don't quite work. Which is 100% okay!